tiatoolbox/tools/registration/prealignment.py

```diff
@@ -0,0 +1,96 @@
+import cv2
+import numpy as np
+import scipy.ndimage as ndi
+from skimage import exposure
+from skimage.util import img_as_float
+
+from tiatoolbox.utils.metrics import dice
+
+
+def prealignment(fixed_img, moving_img, fixed_mask, moving_mask, rotation_steps=10):
+    """Coarse registration of an image pair.
+
+    This function performs rough alignment of a moving image with respect
```

Comment:
```suggestion
    This function performs initial alignment of a moving image with respect to a fixed image. This can be used as a prealignment step before final refinement?
```